packages/ui/src/app/pages/WorkingGroups/WorkingGroup/WorkingGroup.stories.tsx

```diff
@@ -96,3 +151,144 @@ export default {
+
+      await waitFor(() => expect(nextButton).toBeDisabled())
+
+      await userEvent.type(openingTitleField, 'Membership worker role')
+      await userEvent.type(shortDescriptionField, 'Lorem Ipsum...')
+      ;(await getEditorByLabel(modal, 'Description')).setData('Bigger Lorem ipsum...')
+      await waitFor(() => expect(nextButton).toBeEnabled())
+      await userEvent.click(nextButton)
+    })
+    await step('Duration & Process', async () => {
+      await waitFor(() => expect(nextButton).toBeDisabled())
+      ;(await getEditorByLabel(modal, 'Application process')).setData('Application process default')
+      await waitFor(() => expect(nextButton).toBeEnabled())
+      await userEvent.click(modal.getByText('Limited'))
+      const expectedLengthField = modal.getByLabelText('Expected length of the application period')
+      await userEvent.clear(expectedLengthField)
+      await userEvent.type(expectedLengthField, '2000')
+      await waitFor(() => expect(nextButton).toBeEnabled())
+      await userEvent.click(nextButton)
+    })
+    await step('Application Form', async () => {
+      await waitFor(() => expect(nextButton).toBeDisabled())
+      await userEvent.type(modal.getByRole('textbox'), '🐁?')
+      await waitFor(() => expect(nextButton).toBeEnabled())
+      await userEvent.click(modal.getByText('Add new question'))
+      await waitFor(() => expect(nextButton).toBeDisabled())
+      await userEvent.click(modal.getAllByText('Long answer')[1])
+      await userEvent.type(modal.getAllByRole('textbox')[1], '🐘?')
+      await waitFor(() => expect(nextButton).toBeEnabled())
+      await userEvent.click(nextButton)
+    })
+    await step('Staking Policy & Reward', async () => {
+      const createButton = getButtonByText(modal, 'Create Opening')
+      expect(createButton).toBeDisabled()
+      await userEvent.type(modal.getByLabelText('Staking amount *'), '100')
+      await userEvent.clear(modal.getByLabelText('Role cooldown period'))
+      await userEvent.type(modal.getByLabelText('Role cooldown period'), '1000')
+      await userEvent.type(modal.getByLabelText('Reward amount per Block'), '0.1')
+      await waitFor(() => expect(createButton).toBeEnabled())
+      await userEvent.click(createButton)
+    })
+
+    await step('Sign transaction and Create', async () => {
+      expect(await modal.findByText('You intend to create an Opening.'))
+      await userEvent.click(modal.getByText('Sign transaction and Create'))
+    })
+
+    step('Transaction parameters', () => {
+      const [description, openingType, stakePolicy, rewardPerBlock] = args.onCreateOpening.mock.calls.at(-1)
+
+      expect(stakePolicy.toJSON()).toEqual({
+        stakeAmount: 100_0000000000,
+        leavingUnstakingPeriod: 1000,
+      })
+      expect(new BN(rewardPerBlock).toNumber()).toEqual(1000000000)
+
+      expect(openingType).toEqual('Regular')
+      expect(metadataFromBytes(OpeningMetadata, description)).toEqual({ ...WG_OPENING_DATA })
```

Comment:
Small detail but there is not need to spread the object it won't get mutated:
```suggestion
      expect(metadataFromBytes(OpeningMetadata, description)).toEqual(WG_OPENING_METADATA)
```